backend/mora/service/orgunit.py

```diff
@@ -106,6 +106,10 @@ def edit(self, unitid, req):
             original, mapping.PARENT_FIELD.path)[-1]
         parent_uuid = util.get_uuid(parent)
 
+        org = util.get_obj_value(
```

Comment:
Don't we have a `get_obj_uuid` (or something) for exactly this case?

Reply:
We have a ton of `get_*` functions, but not one for this particular purpose, AFAIK.